sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -86,6 +87,9 @@ export default {
       type: String,
       required: true,
     },
+    readOnly: {
+      type: String,
+    },
```

Comment:
did you check, what happens when `readOnly is ""` (when `readOnly` is not passed)?